src/roles/hammer/defaults/main.yml

```
@@ -0,0 +1,9 @@
+---
+hammer_foreman_server_url: "https://{{ ansible_fqdn }}"
+hammer_ca_certificate: "/etc/pki/tls/certs/ca-bundle.trust.crt"
+hammer_packages:
+  - rubygem-hammer_cli
+  - rubygem-hammer_cli_foreman
+  - rubygem-hammer_cli_foreman_tasks
+  - rubygem-hammer_cli_foreman_remote_execution
+  - rubygem-hammer_cli_katello
```

I thought we had implemented Provides for this, but https://github.com/theforeman/foreman-packaging/commit/10e0424442e2a8d9703d83e07801218676a480f6 hasn't propagated to the ones with nightly packages. I'd like to have it clean from the start in the next installer so can we update our packaging and use:
```suggestion
  - hammer-cli-plugin-foreman
  - hammer-cli-plugin-foreman_tasks
  - hammer-cli-plugin-foreman_remote_execution
  - hammer-cli-plugin-katello
```

Later we can also update our Debian packages to do the same.

---

I stand corrected, I tried, and it works!

```
# ansible -m ansible.builtin.package -a name=hammer-cli-plugin-foreman_remote_execution -v localhost
Using /etc/ansible/ansible.cfg as config file
localhost | SUCCESS => {
    "changed": false,
    "msg": "Nothing to do",
    "rc": 0,
    "results": []
}
```